site/profile/templates/puppet.conf.erb

```diff
@@ -0,0 +1,5 @@
+# File managed by Puppet
+[main]
+server = <%= @options['server'] %>
+noop = <%= @options['noop'] %>
+
```

Comment:
Needless to say that if you want yuou can add more options here and set them on hiera via the psick::puppet::tp::options_hash key. Note also that for this key (as for all the psick params whose name ends with _hash) a deep merge lookup is done across the hierarchies